code/modules/goldmansachs/pay_stand.dm

```diff
@@ -0,0 +1,39 @@
+/obj/machinery/paystand
+	name = "Pay Stand ($0)"
+	desc = "Owned by . Pays directly into his account when swiped with an ID card."
+	icon = 'icons/obj/economy.dmi'
+	icon_state = "card_scanner"
+	resistance_flags = INDESTRUCTIBLE|LAVA_PROOF|FIRE_PROOF|ACID_PROOF
+	density = FALSE
+	anchored = TRUE
+	var/price = 0
+	var/obj/item/card/id/my_card
+
+/obj/machinery/paystand/proc/relocate(area/new_area)
+	say("We're moving to a new location at [new_area]! See you capitalist consumers there!")
+	qdel(src)
+
+/obj/machinery/paystand/attackby(obj/item/W, mob/user, params)
+	if(!my_card)
+		say("ERROR: NO OWNER FOUND, SELF DESTRUCTION IMMINENT!!!")
+		qdel(src)
+	if(istype(W, /obj/item/stack/spacecash))
+		say("What is this, the 2000s? We only take card here.")
+		return
+	if(istype(W, /obj/item/coin))
+		say("What is this, the 1800s? We only take card here.")
+		return
+	if(istype(W, /obj/item/card/id))
```

Comment:
Its more likely for someone to use their card so move this above the other 2 checks and change those two checks to else if

Reply:
ye

Reply:
is that a ye i will fix this or